### PR TITLE
Finding prime number in  range.

### LINKED ISSUE #2964: Finding Prime Number in a range from a to b ( Method 1 ) ( nestedloop) ( c++)
## 💥 Proposal

(A clear and concise description of what the proposal is.)

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#reporting-new-issues)?
Yes

(Write your answer here.)
I will like to add code in c++ language which gives prime number  in range a to b given by user !!

### PR DESCRIPTION
**Pull request by RuchiGitHu**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

A positive integer greater than 1 which has no other factors except 1 and the number itself is called a prime number. 2, 3, 5, 7 etc. are prime numbers as they do not have any other factors.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

fixes #2964

### CODE REVIEW

File: Java/cp/Prime_range.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    Input :
+    Please enter range..
+    70
+    80
+
+    Output:
+    71
+    73
+    75
+    77
```
Comment (rohanbabbar04) [inline Java/cp/Prime_range.java:42]: 75 and 79 are not prime numbers
Do correct it

Comment (rohanbabbar04) [inline Java/cp/Prime_range.java]: 75 and 77 are not prime numbers
Do correct it

**Review by rohanbabbar04 — CHANGES_REQUESTED** [review]

File: Java/cp/Prime_range.java (hunk 1 above)
Comment (RuchiGitHu) [reply Java/cp/Prime_range.java]: > 75 and 77 are not prime numbers
> Do correct it

Please review @rohanbabbar04 , I have corrected the code.

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Fix the indentation

**Comment by RuchiGitHu** [comment]
> Fix the indentation

Tried to fix it, please check... @HarshCasper @rohanbabbar04 @ankitaggarwal23

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Format and indent the Code properly.

**Comment by RuchiGitHu** [comment]
> Format and indent the Code properly.

Please check again whenever you can @HarshCasper @ankitaggarwal23 @rohanbabbar04

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Left aligns the whole code. There seems to be an unnecessary indentation

**Comment by RuchiGitHu** [comment]
> Left aligns the whole code. There seems to be an unnecessary indentation

I have tried. Please check @HarshCasper and also let me know that next time how can I format code correctly. I use Atom as a editor but after pushing the code, indentation gets break.
Thanks for reviewing....

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**